github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1164,6 +1165,8 @@ periodics:
         value: "true"
       - name: KUBEVIRT_PROVIDER_EXTRA_ARGS
         value: --prometheus-port 30007
+      - name: KUBEVIRT_WITH_ETC_CAPACITY
```

Comment:
nit: i don't see a default value for the capacity setting in [the preset](https://github.com/kubevirt/project-infra/blob/283f734392890293ca144adff481f4a19be80862/github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml#L638) - should we move this env into the preset? Or maybe add it with the default value so people are more aware that they can override it?